_posts/2017-11-28-data-science-assessment-challenges.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+subheadline:
+title: "Challenges Assessing Data Science"
+Teaser: What is data science? What are the core competencies?
+header:
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["Marianne Corvellec " "Kari L. Jordan "]
+---
+On November 15, the [virtual assessment network](https://groups.google.com/a/carpentries.org/forum/#!forum/assessment-network) held its fifth meeting. It was organized and hosted online by Kari Jordan, and six community members attended. The conversation revolved around data science education. 
```

Comment:
The link to the Google groups goes to a page where you have to be a member to see anything. Is there any other page that describes the virtual assessment network?

Reply:
Right now there's a repo and then our [Assessment page](http://www.datacarpentry.org/assessment/). I could add information about the assessment network to the assessment page, or I could add a new page under the Get Involved tab of the website. I like the second option (page under get involved tab)